client/src/Dashboard.tsx

```diff
@@ -113,10 +128,107 @@ const Dashboard = () => {
           }
         }}
       />
+      <Modal
+        // youtube to playlist converter modal
+        show={isYoutubePlaylistConverterModalShow}
+        onClose={() => {
+          setYoutubePlaylistURL('');
+          setSpotifyPlaylistID('');
+          setIsYoutubePlaylistConverterModalShow(false);
+          setIsConverting(false);
+        }}
+        title={`Youtube Playlist to Spotify Playlist Converter`}
+        content={
+          <>
+            <div>
+              Will create a public playlist in your spotify account from the tracks given in the youtube playlist. The
+              input should be a URL to the youtube playlist. Our team has observed issues with converting youtube
+              playlists that are also 'mixtapes'; it is recommended that only playlists that are not mixtapes are used
+              given to the converter. Note that our team has imposed a limit of roughly 500 songs that can be added to a
+              playlist in a single conversion. If the given youtube playlist has over 500 songs, we will only add the
```

Comment:
This description is kind of long from a UX perspective. You can use a tooltip

Reply:
Another PR

Reply:
I think you can make a shorter, simpler description. It's kind of obvious what it does.
> Converts a YouTube playlist to a playlist in your Spotify account. Maximum 500 songs.

The stuff about mixtapes should be in an info tool-tip.

Reply:
Can you at least make the background and text work with the theme set (dark mode, light mode)